cmd/kuberhealthy/reaper.go

```diff
@@ -40,8 +44,26 @@ func init() {
 		cfg.MaxCheckPods = maxCheckerPodsDefault
 	}
 
-	if cfg.JobCleanupDuration == 0 {
-		cfg.JobCleanupDuration = jobCleanupDurationDefault
+}
+
+// parseDuration parses checkReaper duration configs into time.Duration. Go parses time.Duration from int64 and not from
+// string, which is how the config reloader unmarshals the config json. This way we can still supply time.Duration type
+// values in the config, instead of an int64 that doesn't allow users to specify the duration type.
+func parseDuration() {
+
+	var err error
+	if len(cfg.JobCleanupDuration) != 0 {
+		jobCleanupDuration, err = time.ParseDuration(cfg.JobCleanupDuration)
+		if err != nil {
+			log.Errorln("error occurred attempting to parse JobCleanupDuration:", err, "Using default value:", jobCleanupDuration)
```

Comment:
In both cases where we run `time.ParseDuration` here, we should be certain what is returned when an error occurs.  If `time.ParseDuration` returns a `0`, this could cause some real chaos.  Consider setting the default explicitly after an error occurs.